### ISSUE TITLE
[Accessibility] Unable to access Status bar controls using keyboard

### ISSUE THREAD

**Issue by setaskin** [microsoft/vscode#120211]
- VS Code Version: 1.54.3  stable 
- OS Version: Windows

Steps to Reproduce:

1. open Vs code
2. select any tree item ( "search" or "explorer" etc.)
3.  try to navigate via "tab" key
 actual : keyboard only users cannot use status bar items. 

![image](https://user-images.githubusercontent.com/37163639/113076268-8307c580-9183-11eb-8efd-a0a373fcbb35.png)

Does this issue occur when all extensions are disabled?: Yes/

**Comment by setaskin** [comment]
@isidorn , this is another issue that is opened by accessibility test team for Live Share

**Comment by gjsjohnmurray** [comment]
#94677 indicates this is intentional.

#94921 covers F6 being added as a way to navigate to the statusbar.

**Comment by isidorn** [comment]
@gjsjohnmurray thanks for answering.
@setaskin yes, as @gjsjohnmurray pointed out this is intentional and based on user feedback.
Status bar can be focused via F6 and the status bar can be read out using the special read status bar commands.
Just to be sure I have raised this question one more time with our screen reader community but for now no plans to change this

**Comment by isidorn** [comment]
After discussing with the community we decided to make the status bar a tab stop and to add a command to focus on the status bar.

**Comment by isidorn** [comment]
I have pushed a fix for this after discussing with the community:
1) Status bar is now a tab stop
2) introduced a command "Focus Status Bar"
3) Pressing `Esc` while focus is on status bar moves focus back to editor

fyi @jvesouza @webczat @MarcoZehe

**Comment by isidorn** [comment]
I have heard from the community in our gitter channel that this works nicely.
However please let me know if there is additional feedback.

**Comment by jvesouza** [comment]
Works ok for me.
Thanks.
